### PR TITLE
Consolidate test utils under `tests/utils`

### LINKED ISSUE #2871: [tests] Consolidate test utils under tests package
* FilesystemTestRig
* assertGoldensMatch

### PR DESCRIPTION
**Pull request by kevinpschaaf**
Moves the following utilities to `tests/utils`:
* FilesystemTestRig (from wireit, modified to accept path parts and use a temp dir, otherwise the same)
* assertGoldensMatch (extracted from localize-tools)

Fixes #2871

### CODE REVIEW

**Review by aomarks — APPROVED** [review]
Nice!

**Comment by kevinpschaaf** [comment]
@aomarks Sorry just pushed up one more change to add wireit to the tests/utils build, mind checking that?

**Review by aomarks — APPROVED** [review]

**Review by aomarks — APPROVED** [review]

File: packages/labs/cli/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "eslint": "^5.16.0",
     "globby": "^10.0.2",
     "stdout-stderr": "^0.1.13",
+    "tests": "^0.0.0",
```
Comment (justinfagnani) [inline packages/labs/cli/package.json:39]: Not now, but: to make things clear, I'd like to move internal packages to under a scope, like `@lit-internal` or something.

**Review by justinfagnani — APPROVED** [review]

File: packages/labs/cli/package.json (hunk 1 above)
Comment (kevinpschaaf) [reply packages/labs/cli/package.json:39]: good idea

**Pull request merged**